steps/upload-assets-to-github-release/1.0.1/step.yml

```diff
@@ -0,0 +1,75 @@
+title: Upload assets to Github Release
+summary: |
+  Upload assets to Github Release
+description: |
+  Attach asset(s) from your build to an existing Github Release. When building a tag, the Release will be looked via that tag.
+  By default, it will try and cache all Android and iOS artifacts.
+website: https://github.com/jasperroel/bitrise-step-upload-assets-to-github-release
+source_code_url: https://github.com/jasperroel/bitrise-step-upload-assets-to-github-release
+support_url: https://github.com/jasperroel/bitrise-step-upload-assets-to-github-release/issues
+published_at: 2019-12-19T16:43:34.263689+01:00
+source:
+  git: https://github.com/jasperroel/bitrise-step-upload-assets-to-github-release.git
+  commit: df7301f790640a3f716bef3a19516a833191a57b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: hub
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- debug: false
+  opts:
+    description: |
+      Enables "hub" debug output
+    is_expand: true
+    is_required: false
+    summary: |
+      Enables "hub" debug output
+    title: Debug this step
+    value_options:
+    - "true"
+    - "false"
+- github_pat: null
```

Comment:
Could name this input `personal_access_token` no need to include github in the name as it is in the title.